### PR TITLE
Add Connector Salesforce

### PR DESCRIPTION
**Pull request by gurneyalex**
port of https://github.com/OCA/connector-interfaces/pull/11 to this repository.

### CODE REVIEW

File: connector_salesforce/README.rst [hunk 1]
```diff
@@ -0,0 +1,247 @@
+Salesforce Connector
```
Comment (yvaucher) [inline connector_salesforce/README.rst]: Please add license badge

File: connector_salesforce/README.rst [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+ - A way must be found in order not to reinvent the wheel with SO onchange management
+   without depending on connector ecommerce addons
+ - Fix simplesalesforce exception lookup on next release of library
+ - Import Quotation must copy the address and store it in deactivate state
+ - Support the usecase of deactivating a deleted binding
+ - Battle test password flow and fix it
+
+Credits
+=======
+* Nicolas Bessi
```
Comment (yvaucher) [inline connector_salesforce/README.rst]: Could you already add Contributors, and add your email as on template ?

File: connector_salesforce/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{'name': 'Salesforce Connector',
+ 'version': '0.1',
+ 'author': 'Camptocamp',
```
Comment (yvaucher) [inline connector_salesforce/__openerp__.py:23]: As @gurneyalex said please add "Odoo Community Association (OCA)" to the authors.

File: connector_salesforce/backend.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Nicolas Bessi
+#    Copyright 2014 Camptocamp SA
```
Comment (yvaucher) [inline connector_salesforce/backend.py:5]: 2015 ?

File: connector_salesforce/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Nicolas Bessi
```
Comment (yvaucher) [inline connector_salesforce/__openerp__.py:4]: Missing copyright mention

File: connector_salesforce/__init__.py [hunk 6]
```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Nicolas Bessi
+#    Copyright 2014 Camptocamp SA
```
Comment (yvaucher) [inline connector_salesforce/__init__.py]: 2015 ? or 2014-2015 ?

File: connector_salesforce_server_environment/__openerp__.py [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{'name': 'Connector Salesforce Server Environment',
+ 'version': '0.1',
+ 'author': 'Camptocamp',
```
Comment (yvaucher) [inline connector_salesforce_server_environment/__openerp__.py]: Please add OCA in authors

File: connector_salesforce/__openerp__.py (hunk 5 above)
Comment (gurneyalex) [inline connector_salesforce/__openerp__.py:4]: Fixed

**Comment by gurneyalex** [comment]
@guewen a final review (since quite a lot of time has passed since the initial submission of the PR) would be welcome.

**Comment by gurneyalex** [comment]
runbot is orange due to an unrelated warning.

**Comment by pedrobaeza** [comment]
@gurneyalex are you still interested in this?

**Comment by pedrobaeza** [comment]
No answer, so I close it. Feel free to reopen if needed.

**Pull request closed without merging**